static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -476,10 +596,6 @@ const TaskWrapper = styled(motion.li)`
   gap: ${space(1)};
 `;
 
-TaskWrapper.defaultProps = {
-  layout: true,
-};
-
```

Comment:
Removing this because the layout currently animates everything, including height changes. With the new skip confirmation displayed below the task, the animation no longer looks good. I’ll add a better animation as part of [this task](https://github.com/getsentry/projects/issues/337).